qt/widgets/spectrumviewer/CMakeLists.txt

```diff
@@ -1,84 +1,77 @@
-set ( SRC_FILES
-       src/ColorMaps.cpp
-       src/SpectrumDisplay.cpp
-       src/SVUtils.cpp
-       src/TrackingPicker.cpp
-       src/DataArray.cpp
-       src/SpectrumPlotItem.cpp
-       src/QtUtils.cpp
-       src/GraphDisplay.cpp
-       src/SpectrumView.cpp
-       src/SliderHandler.cpp
-       src/RangeHandler.cpp
-       src/EModeHandler.cpp
-       src/SpectrumDataSource.cpp
-       src/SVConnections.cpp
-       src/MatrixWSDataSource.cpp
-       #src/MatrixWSSpectrumView.cpp
-       src/ArrayDataSource.cpp
-)
+set(SRC_FILES
+    src/ColorMaps.cpp
+    src/SpectrumDisplay.cpp
+    src/SVUtils.cpp
+    src/TrackingPicker.cpp
+    src/DataArray.cpp
+    src/SpectrumPlotItem.cpp
+    src/QtUtils.cpp
+    src/GraphDisplay.cpp
+    src/SpectrumView.cpp
+    src/SliderHandler.cpp
+    src/RangeHandler.cpp
+    src/EModeHandler.cpp
+    src/SpectrumDataSource.cpp
+    src/SVConnections.cpp
+    src/MatrixWSDataSource.cpp
+    # src/MatrixWSSpectrumView.cpp
```

Comment:
Check this one with martyn, the file does exist, and I couldn't see a duplication here.

Reply:
These file have been left around by accident so I've removed these comments and deleted the files (with Martyn's permission)